doc/developer-guides/coding_guidelines.rst

```diff
@@ -3141,6 +3141,100 @@ Compliant example::
        // This is a comment
 
 
+CS-18: Function information shall be documented with doxygen-style comments
+===========================================================================
+
+Some detailed rules are listed below to illustrate the comments format for each
+function:
+
+1) The comments block shall start with '/\*\*'(slash-asterisk-asterisk) in a
```

Comment:
Wouldn't it be better to enclose code sequences in double-backticks, i.e. ` ``/**`` `

(same comment throughout the doc)

Reply:
Because of formatting issues with spaces in the inline blocks, we'll not use them in this section.